install_config/topics/vsphere_prerequisites.adoc

```diff
@@ -3,128 +3,136 @@
 // * install_config/configuring_for_vsphere.adoc
 
 [[vsphere-prereqs]]
-= VMware vSphere cloud provider prerequisites
+=== Requirements
 
-.Prerequisites
+VMware vSphere
+(*Note:* Standalone ESXi is not supported.)
 
-Enabling VMware vSphere requires installing the VMware Tools on each Node VM.
+* vSphere version 6.0.x minimum recommended version 6.7 U1b is required if you intend to support a complete link:https://docs.vmware.com/en/VMware-Validated-Design/5.0.1/rn/vmware-validated-design-501-release-notes.html[VMware Validate Design].  
+* vSAN, VMFS and NFS supported.
+** vSAN support is limited to one cluster in one vCenter.
+
+=== Prerequisites
+
+VMware vSphere requires installation of the VMware Tools on each Node VM.
 See
 link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.html.hostclient.doc/GUID-ED3ECA21-5763-4919-8947-A819A17980FB.html[Installing VMware tools] for more information.
 
-.Procedure
+The link:https://github.com/vmware/govmomi/tree/master/govc#govc[govc] CLI can be used for additional configuration and troubleshooting. Example govc configuration:
```

Comment:
We don't allow new GitHub links unless you have explicit permission from support.

Reply:
Ok how do you get this approved for just docs ?  @dav1x ?

Reply:
@sferich888, are you ok with this GitHub link?

Reply:
I think I would prefer lines ~157-180 be removed from this PR. 

There are several reasons for this. 

1. Searching for govc on Google, https://www.google.com/search?q=govc does not show me or take me to an official VShphere domain, on the fist page. 

As a result I don't see this as an official, default or well used tool in the VMware community. 

2. The process, if this is the first time using this tool, is more complicated (5+ steps) over the ~5 clicks in the UI. 

3. The UI, the default process we're pushing in these docs, is simple and more likely to be what customer use. 

With all of that said, I learned something new (cool tool). I wonder if it's use and or highlights would be better suited in a blog post.

Reply:
Thank you @sferich888! @RobbieJVMW, please make the updates that Eric's requested.

Reply:
@sferich888 - the first hit with I google the term is the office VMware GitHub site.  The govc tool is part of the VMware vSphere SDK (govmomi being the full name for the tool) and used extensively both inside VMware and externally; As it is open source we host it on GitHub under the VMware Opensource repo, not VMware.com.

Would it be more helpful to clarify rather than remove the link? 
With all that said my expectation is those vSphere admins working in this area would know the tool (or the PowerShell equivalent) so happy to remove the link and simply mention the tool instead.

Reply:
@kalexand-rh @sferich888 Any further feedback ?

Reply:
@sferich888 Thank you for your feedback!

I did a small cover on govc around OCP 3.6 time frame in a blog post here:
https://developers.redhat.com/blog/2017/10/24/vsphere-cloud-provider-vmdk-dynamic-provisioning/

But, I would love to revisit the topic.

Reply:
The only problem I have with this call out is that it's not explicitly clear that Red Hat does not support the tool, that it's a community tool, and that VMware may or may not support the tool if a customer were to use it. Nore do we explain how a customer gets a hold of VMware or the VMware community to get support for the tool/procedure. 

If we can clarify this in some way, we can leave the links and procedure.